src/bin/cli.ts

```diff
@@ -4,7 +4,7 @@ import * as yargs from 'yargs';
 import { bundle } from '../bundle';
 
 const main = async () => {
-  const { entries, outdir, node, includeAwsSdk, artifactPrefix } = await yargs(
```

Comment:
Any chance we could leave this in? I really like the option to exclude the sdk

Reply:
I like this idea - https://github.com/lifeomic/blamda/pull/17#issuecomment-1744320140.  AWS intetionally ships an SDK so that lambdas don't need to. We should use it.